src/Hazelcast.Net.Tests/Clustering/MemberConnectionTests.cs

```diff
@@ -128,7 +128,7 @@ public async Task Test()
             {
                 await AssertEx.ThrowsAsync<TaskTimeoutException>(async () =>
                 {
-                    await invocation.WaitRetryAsync(correlationIdSequence.GetNext);
+                    await invocation.WaitRetryAsync(correlationIdSequence.GetNext, CancellationToken.None);
```

Comment:
Why not provide default value of `None` for `CancellationToken`? I think it's quite common approach.

Reply:
Make sense, fixing.